src/mixins/filing-template-mixin.ts

```diff
@@ -376,16 +366,24 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
       },
       changeOfRegistration: {
         business: {
-          natureOfBusiness: '',
-          identifier: this.getBusinessId
+          identifier: this.getBusinessId,
+          legalType: this.getEntityType
         },
         contactPoint: this.getContactPoint
       }
     }
 
-    // Apply name request info to filing
+    // Apply NAICS change to filing
```

Comment:
This just got moved up a bit, so it's closer to the main correction object and business sub-object that it updates.